common/user/index.ts

```diff
@@ -266,3 +267,19 @@ export async function queryUser<Select extends UserSelect>(user: User, select: S
 
     throw new Error(`Unknown User(${user.userID})`);
 }
+
+export async function updateUser(userId: string, email?: string, update?: Partial<User>) {
```

Comment:
{ email: string } as second parameter and remove unused update ... We might want to extend this for other properties in the future